### PR TITLE
change doc order # 774

### PR DESCRIPTION
**Pull request by lijiarui**
#774 

- [x] tune the order of APIs
- [ ] change the links inside README.md from Wiki to /docs/index.md
- [X] `@private` all deprecated APIs.

Next step: After you merge this, I will add friend request doc

### CODE REVIEW

File: src/wechaty.ts [hunk 1]
```diff
@@ -1,19 +1,18 @@
 /**
- *   Wechaty - https://github.com/chatie/wechaty
  *
  *   @copyright 2016-2017 Huan LI <zixia@zixia.net>
  *
- *   Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (zixia) [inline src/wechaty.ts]: Why do you change the comment format at here?

Do not include unrelated changes.

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Wechaty is a Bot Framework for Wechat **Personal** Account which can help you cr
 
 :octocat: <https://github.com/chatie/wechaty>  
 :beetle: <https://github.com/chatie/wechaty/issues>  
-:book: <https://github.com/chatie/wechaty/wiki>  
+:book: <https://github.com/Chatie/wechaty/blob/master/docs/index.md>
```
Comment (zixia) [inline README.md]: Please use the url <http://chatie.io/wechaty/> instead of https://github.com/Chatie/wechaty/blob/master/docs/index.md> at here and all of the following links in the document.

Because <http://chatie.io/wechaty/> is the custom url of <https://github.com/Chatie/wechaty/blob/master/docs/>

**Comment by zixia** [comment]
Could you please make sure that you are just moving code blocks, without make any additional modifications?

**Comment by lijiarui** [comment]
Also add `@private all deprecated APIs.`

how do you think about it?

**Comment by zixia** [comment]
That's OK. Thanks!
